dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureReplacementAction.java

```diff
@@ -417,6 +383,45 @@ public void setup() throws Exception {
             LibertyClientFactory.getLibertyClient(clientName);
     }
 
+    protected void swizzleFeatures(Set<String> features) {
```

Comment:
Looks like no change in the code here, just breaking it out into its own method right?

Reply:
Suggested name change: `swizzelFeatures` --> `replaceFeatures`

Reply:
Right - I had originally moved that to its own method in order to extend only that functionality.  If we make the change to the base class as you suggested later, then we no longer to split it out.